test/fixtures/SpokePool.Fixture.ts

```diff
@@ -74,22 +82,23 @@ export async function deposit(
   recipient: SignerWithAddress,
   depositor: SignerWithAddress,
   destinationChainId: number = consts.destinationChainId,
-  amountToDeposit: BigNumber = consts.amountToDeposit,
-  depositRelayerFeePct: BigNumber = consts.depositRelayerFeePct,
-  quoteTimestamp?: number
+  amount = consts.amountToDeposit,
+  relayerFeePct = consts.depositRelayerFeePct,
+  quoteTimestamp?: number,
+  message?: string
 ) {
-  await spokePool
-    .connect(depositor)
-    .deposit(
-      ...getDepositParams(
-        recipient.address,
-        token.address,
-        amountToDeposit,
-        destinationChainId,
-        depositRelayerFeePct,
-        quoteTimestamp ?? (await spokePool.getCurrentTime())
-      )
-    );
+  quoteTimestamp ??= await spokePool.getCurrentTime();
+  await spokePool.connect(depositor).deposit(
```

Comment:
This looks incorrect - it should probably use `BigNumber.toNumber()` instead.

Reply:
Fixed: https://github.com/across-protocol/contracts-v2/pull/363/commits/217dc9ea8508e0e90f440c5ddfd97a4d9c8b875f